test/e2e/windows/hybrid_network.go

```diff
@@ -24,6 +24,7 @@ import (
 	"k8s.io/apimachinery/pkg/util/uuid"
 	"k8s.io/kubernetes/test/e2e/framework"
 	e2eskipper "k8s.io/kubernetes/test/e2e/framework/skipper"
+	"k8s.io/kubernetes/test/e2e/network"
```

Comment:
why do we want to add this dependency?

Reply:
I added this dependency because I thought that we needed func [`checkConnectivityToHost`](https://github.com/kubernetes/kubernetes/blob/master/test/e2e/network/networking.go#L44) for checking the internet connectivity, but since we are using func [`assertConsistentConnectivity`](https://github.com/kubernetes/kubernetes/blob/master/test/e2e/windows/hybrid_network.go#L91) then we don't need it. 
Making appropriate changes.

Reply:
yeah, sorry, I was going through the PR and started adding the comments, most of them are duplicates :)